com.unity.render-pipelines.universal/Runtime/2D/Passes/Render2DLightingPass.cs

```diff
@@ -11,9 +11,10 @@ internal class Render2DLightingPass : ScriptableRenderPass
         Renderer2DData m_Renderer2DData;
         static readonly ShaderTagId k_CombinedRenderingPassNameOld = new ShaderTagId("Lightweight2D");
         static readonly ShaderTagId k_CombinedRenderingPassName = new ShaderTagId("Universal2D");
+        static readonly ShaderTagId k_UnlitPassName = new ShaderTagId("Universal2DUnlit");
```

Comment:
Is this new light mode tag ever used in any of the new or updated shaders? A solution scope search didn't show anything.